Мягкие игрушки для трапов, UWU

polerno opened the pull request:
я сделал шесть разных игрушек, как плюшевых лис. Так и плюшевых скреллов P.S ПОЖАЛУЙСТА, НЕ ЕБИТЕ ПЛЮШЕВЫХ СКРЕЛЛОВ


### Описание изменений
Это нихуя не изменяет кроме добавления прикольных игрушек, скрин с ними сделаю скорее всего завтра

### Почему и что этот ПР улучшит
Вы сможете подарить эту милую игрушку своей вайфу, сделайте свою жизнь ярче

### Авторство
Главный кодер - Foler the third
Спрайтер - хуй знает, я спиздил их

### Чейнджлог

polerno committed:
Мягкие игрушки для трапов, UWU

я сделал шесть разных игрушек, как плюшевых лис. Так и плюшевых скреллов P.S ПОЖАЛУЙСТА, НЕ ЕБИТЕ ПЛЮШЕВЫХ СКРЕЛЛОВ

polerno commented:
@Evie7056, слышь. смотри

polerno commented:
![Снимок](https://user-images.githubusercontent.com/48944552/221421804-d93303a6-5794-47a3-a1e0-4f9f9b56b122.PNG)

пиздатые лисики

ActuallyALovelyGrave commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+/obj/item/toy/figure/plush/skrell
+	icon = 'proxima/icons/obj/toys.dmi'
+	name = "blue plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says: NOT TO USE AS A TOY FOR JOY"
```
`her` -> `it`, toys have only neutral gender since they are soulless, as the person writing this quote.
`next to it it says` -> `next to it says` since piece of paper is the noun of objection. Same applies to every other descriptions

ActuallyALovelyGrave commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+
+/obj/item/toy/figure/plush/skrell/orange
+	name = "Orange plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says:NOT TO USE AS A TOY FOR JOY"
```
`says:NOT` - > `says: NOT` missing space between : and 'NOT'

ActuallyALovelyGrave commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+
+/obj/item/toy/figure/plush/skrell/red
+	name = "Red plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says:NOT TO USE AS A TOY FOR JOY."
```
says:NOT - > says: NOT missing space between : and 'NOT', same.

ActuallyALovelyGrave commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+/obj/item/toy/figure/plush/fox
+	icon = 'proxima/icons/obj/toys.dmi'
+	name = "Plush red fox"
+	desc = "Cute and soft plush fox, it's so nice to hug him. After all, he is fluffy like a real fox."
```
`him` -> `it`.
`he` - > `it` both toys and foxes have neutral genders unless pets, since they are somehow tend to have female/male genders in the speech. Weirdos.

polerno committed:
Update toys.dm

Я ненавижу душнил по английскому, так что даггер. Ты сам будешь писать мне текст на английском

polerno commented:
@ActuallyALovelyGrave, мистер душнила. Текст исправлен, можете проверять

ActuallyALovelyGrave commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+
+/obj/item/toy/figure/plush/skrell/orange
+	name = "Orange plush skrell"
+	desc = "A soft plush toy is so nice to hug it, but on a piece of paper next to it says: PLEASE DON'T USE LIKE SEX TOY"
```
TOY" - > TOY." missing dot.

ActuallyALovelyGrave commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+/obj/item/toy/figure/plush/skrell
+	icon = 'proxima/icons/obj/toys.dmi'
+	name = "blue plush skrell"
+	desc = "A soft plush toy is so nice to hug it, but on a piece of paper next to it says: PLEASE DON'T USE LIKE SEX TOY"
```
TOY" - > TOY." missing dot.

polerno committed:
Update toys.dm

Исправлено

polerno committed:
Update toys.dm

polerno committed:
Update toys.dm

ghost commented:
Done, fixed.

ActuallyALovelyGrave reviewed (approved):
Done fixed bla-bla-bla.

HaCsO commented on proxima/code/game/items/toys.dm:
```diff
@@ -0,0 +1,31 @@
+	desc = "Cute and soft plush fox, it's so nice to hug it. After all, it is fluffy like a real fox."
+	icon_state = "redfox"
+
+/obj/item/toy/figure/plush/fox/niggerfox
```
nigga

polerno replied:
так надо, не пали контору

polerno commented:
![Снимок](https://user-images.githubusercontent.com/48944552/221426572-e67b4e8a-869d-4d53-ace1-ccd53caf20ff.PNG)
![Снимок1](https://user-images.githubusercontent.com/48944552/221426574-5515c632-5210-4e71-acf9-f0cd9c750d88.PNG)
НЕ ТРАХАТЬ СКРЕЛЛОВ И ЛИСИЧЕК

johncoder0 commented:
господи как я хочу трахать плюшевых скреллов

Evie7056 commented:
@polerno закинь в лоадаут в `plush toy` селектор

polerno commented:
> @polerno закинь в лоадаут в `plush toy` селектор

Я ХОЧУ ОТГРЫЗТЬ ТЕБЕ ЕБАЛО

polerno committed:
ЛОДАУТ ЁПТА

добавляем игрушки в лодаут, ева если ты это читаешь. Я тебя ебал в сраку
